account_voucher_tax/account_statement_bank.py

```diff
@@ -21,12 +21,13 @@
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
 #
 ##############################################################################
+from __future__ import division
```

Comment:
This change may modify the result of some methods here. Are there unit tests to verify the results?
We have had problems in other instances when we have added this change.
Let me know if you consider that there is no problem with this change.